R/tune_race_anova.R

```diff
@@ -22,6 +22,10 @@
 #' @param metrics A [yardstick::metric_set()] or `NULL`.
 #' @param control An object used to modify the tuning process. See
 #'  [control_race()] for more details.
+#' @param eval_time A numeric vector of time points where dynamic event time
+#' metrics should be computed (e.g. the time-dependent ROC curve, etc). The
+#' values should be non-negative and should probably be no greater then the
+#' largest event time in the training set.
```

Comment:
```suggestion
#' values must be non-negative and should probably be no greater than the
#' largest event time in the training set.
```

How important do you consider this caution about "greater than the largest event time"? I think we should either add an explanation as to why (can be as short as "it's extrapolating") or remove it. If we want to keep it, and it gets a bit long, we can put it in the Details section. In the context of the argument definition a description with "should probably" seems too vague.

Reply:
I _think_ that this limitation is due to KM curves, which affects the probability of censoring and other aspects. It is extrapolation, but I think that the main impediment is related to "cannot compute" more than anything. 

I agree about the vagueness.

Reply:
I've been playing with the censoring model and yes, predictions of censoring probability for "out of range" times, i.e. times greater than the largest event time, are `NA` (but the prediction itself doesn't error).